vsc_pdf_template_editor/lib/src/stores/tree_store.dart

```diff
@@ -106,10 +120,11 @@ abstract class TreeStoreModel with Store {
           _templateController.text.substring(cursorPos);
       _templateController.text = newMap;
       _template = jsonDecode(newMap);
-      _buildPdf();
       buildErrorText = '';
+      _buildPdf();
     } catch (e, s) {
       buildErrorText = '$e \n $s';
+      debugPrint('$e\n$s');
```

Comment:
Here I'm also logging to the debug console so we can quickly jump to the location in the stacktrace.